### PR TITLE
Charlie error report curation

### PR DESCRIPTION
**Pull request by cbarcl01**
Minor edits to fix issues identified in error report including, addition of parent classes to definitions to resolve duplicates, capitalising definitions and resolving duplicate labels.

### CODE REVIEW

File: src/ontology/genepio-edit.owl [hunk 1]
```diff
@@ -4370,7 +4370,7 @@ AnnotationAssertion(obo:GENEPIO_0000001 obo:GENEPIO_0000023 "Collection")
 AnnotationAssertion(obo:GENEPIO_0000004 obo:GENEPIO_0000023 "Received date")
 AnnotationAssertion(obo:GENEPIO_0000005 obo:GENEPIO_0000023 "NML Labware LIMS")
 AnnotationAssertion(obo:IAO_0000112 obo:GENEPIO_0000023 "year/month/day")
-AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0000023 "date lab received isolate")
+AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0000023 "A date which describes when the laboratory received the isolate."@en)
```
Comment (ddooley) [inline src/ontology/genepio-edit.owl:4373]: Too picky? "A date which describes when **a** laboratory received **an** isolate."

File: src/ontology/genepio-edit.owl [hunk 2]
```diff
@@ -4234,7 +4234,7 @@ SubClassOf(obo:GENEPIO_0000008 obo:GENEPIO_0002075)
 # Class: obo:GENEPIO_0000009 (other (metadata value))
 
 AnnotationAssertion(obo:IAO_0000114 obo:GENEPIO_0000009 obo:IAO_0000428)
-AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0000009 "\"other (metadata choice)\" indicates that for the given categorical variable, a respondent has chosen an other or additional response not listed."@en)
+AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0000009 "A data item \\\"other (metadata choice)\\\" that indicates for the given categorical variable, a respondent has chosen an other or additional response not listed."@en)
```
Comment (ddooley) [inline src/ontology/genepio-edit.owl:4237]: Its not really needed to repeat the "other (metadata choice)" data item label in the definition."

File: src/ontology/genepio-edit.owl [hunk 3]
```diff
@@ -4864,7 +4864,7 @@ SubClassOf(obo:GENEPIO_0000065 obo:OBI_0000968)
 
 # Class: obo:GENEPIO_0000066 (DuPont identification pattern library)
 
-AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0000066 "a library of RiboPrinter recognised patterns")
+AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0000066 "A dataset library of RiboPrinter recognised patterns.")
```
Comment (ddooley) [inline src/ontology/genepio-edit.owl:4867]: A dataset **which is a** library ...

File: src/ontology/genepio-edit.owl [hunk 4]
```diff
@@ -4937,7 +4937,7 @@ AnnotationAssertion(obo:GENEPIO_0000001 obo:GENEPIO_0000072 "Sequencing")
 AnnotationAssertion(obo:GENEPIO_0000004 obo:GENEPIO_0000072 "Workflow")
 AnnotationAssertion(obo:GENEPIO_0000005 obo:GENEPIO_0000072 "Derived from Sample Sheet")
 AnnotationAssertion(obo:IAO_0000112 obo:GENEPIO_0000072 "Resequencing, FastQ only")
-AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0000072 "the workflow is the pre-defined sequence of steps to run the automated sequencing pipeline.")
+AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0000072 "The workflow is the pre-defined sequence of steps to run the automated sequencing pipeline.")
```
Comment (ddooley) [inline src/ontology/genepio-edit.owl:4940]: A workflow which is a ...

File: src/ontology/genepio-idranges.owl [hunk 5]
```diff
@@ -1,6 +1,6 @@
 Prefix: rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
 Prefix: idsfor: <http://purl.obolibrary.org/obo/IAO_0000598>
-Prefix: dc: <http://purl.org/dc/elements/1.1/>
+Prefix: dc: <http://purl.org/dc/terms/>
```
Comment (ddooley) [inline src/ontology/genepio-idranges.owl:3]: OBO foundry has started (or has been in some cases) been using a slightly longer dcterms: prefix for http://purl.org/dc/terms/

Prefix(dcterms:=<http://purl.org/dc/terms/>)

**Review by ddooley — CHANGES_REQUESTED** [review]

File: src/ontology/genepio-edit.owl (hunk 2 above)
Comment (cbarcl01) [reply src/ontology/genepio-edit.owl:4237]: Thanks, will amend in following commit.

File: src/ontology/genepio-edit.owl (hunk 1 above)
Comment (cbarcl01) [reply src/ontology/genepio-edit.owl:4373]: Nope that definitely sounds better :)

File: src/ontology/genepio-edit.owl (hunk 4 above)
Comment (cbarcl01) [reply src/ontology/genepio-edit.owl:4940]: Thanks!

File: src/ontology/genepio-edit.owl (hunk 3 above)
Comment (cbarcl01) [reply src/ontology/genepio-edit.owl]: Amended in following commit.

File: src/ontology/genepio-idranges.owl (hunk 5 above)
Comment (cmrn-rhi) [reply src/ontology/genepio-idranges.owl]: Resolved commit 64be85a775fb3be6611e4dc9e1ef4d89c1123f94

**Comment by cmrn-rhi** [comment]
@ddooley ready for your re-review / approval.

**Comment by cbarcl01** [comment]
@ddooley I've resolved the comments in a follow-up commit, let me know if there is any additions you require, otherwise are we OK to merge? Thanks for your help and feedback!

**Review by ddooley — APPROVED** [review]

**Pull request merged**